tests/hash-backend-integration/package.json

```
@@ -13,8 +13,6 @@
     "test:integration": "vitest --run"
   },
   "dependencies": {
```

`@apps/hash-api` is actually a dependency 😄 
```suggestion
  "dependencies": {
    "@apps/hash-api": "0.0.0-private",
```